platforms/android/app/src/main/java/com/cci/DownloadExpansionFile.java

```diff
@@ -52,13 +52,18 @@
      * application is using LVL for licensing, it may make sense to eliminate
      * these checks and to just rely on the server.
      */
-    public static final XAPKFile xAPK = new
-
-            XAPKFile(
-            true, // true signifies a main file
-            2, // the version of the APK that the file was uploaded
-            2063994466L // the length of the file in bytes
-    );
+    public static final XAPKFile[] xAPKs = {
+            new XAPKFile(
+                    true, // true signifies a main file
+                    3, // the version of the APK that the file was uploaded
+                    2030178927L // the length of the file in bytes
+            ),
+            new XAPKFile(
+                    false,
+                    0,
+                    4910L

```

Comment:
Is this needed?

Reply:
Just keeping the file name version zero also works.